tests/unit/frbc/frbc_fill_level_target_profile_test.py

```diff
@@ -0,0 +1,74 @@
+from datetime import timedelta, datetime, timezone as offset
+import json
+from unittest import TestCase
+import uuid
+
+from s2wsjson.common import *
+from s2wsjson.frbc import *
+
+
+class FRBCFillLevelTargetProfileTest(TestCase):
+    def test__from_json__happy_path_full(self):
+        # Arrange
+        json_str = """
+{
```

Comment:
I think we should add a couple of tabs in this (or more general in the test generation script), if it's not too much of a change.

Reply:
As it is a multiline string, any tabs will show up in the `json_str`. Json loader should not mind it but it does change the value.
Fixing in seperate ticket: #25